en/components/toc.yml

```diff
@@ -876,13 +882,13 @@
   items:
     - name: Virtual Drop Down
       href: drop_down_virtual.md
-      new: true
+      new: false
 - name: Select
   href: select.md
   new: true
 - name: Autocomplete
   href: autocomplete.md
-  new: true
+  new: false
```

Comment:
"Select" should be "true" - https://www.infragistics.com//angularsite/components/select.html#select-header--footer
We have new "header" and "Footer"